documentation/API_documentation/device-reachability-status-subscriptions-API-Readiness-Checklist.md

```diff
@@ -1,12 +1,12 @@
 # API Readiness Checklist
```

Comment:
For #13 update row to link to https://lf-camaraproject.atlassian.net/wiki/x/JAFmBQ